plugins/lang-plan-a/src/main/java/org/elasticsearch/plan/a/PlanALexer.g4

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
So they should go to `src/main/whatever` if we don't want to include them in the final jar. But not in `src/main/java` IMO.